src/qrl/core/TransactionPool.py

```diff
@@ -15,6 +15,17 @@
 
 
 class TransactionPool:
+    """
+    As each Transaction comes in over the network, it is first stored in
+    TransactionPool.pending_tx_pool before the node validates them. After the TX
+    has been validated, it is moved to TransactionPool.transaction_pool.
+
+    The node pulls from this Pool in order to fill a block, which it then
```

Comment:
1. Node pulls what from this Pool? Better to mention it pulls unconfirmed transaction from the pool.